libs/shared/ui/chart/src/lib/chart/chart.component.ts

```diff
@@ -33,7 +32,5 @@ export class ChartComponent implements OnInit {
       columnNames: ['period', 'close'],
       options: { title: `Stock price`, width: '600', height: '400' }
     };
-
-    this.data$.subscribe(newData => (this.chartData = newData));
```

Comment:
no need to subscribe for observable for sake of just binding to component properties. better to handle in template itself using async pipe. 
Also unsubscribe/complete is not handled on destroy